nlmod/dims/layers.py

```diff
@@ -2269,6 +2266,50 @@ def get_max_overlap_model_layer(i):
     return modellayer
 
 
+def map_points_to_grid(x, y, ds=None, modelgrid=None):
```

Comment:
maybe rename to get_cellids_from_xy(), and move to grid.py?

Reply:
Good idea.